windows/install.cmd

```diff
@@ -60,23 +60,23 @@ WHERE ttx /q || (
 PUSHD %TEMP%
 IF EXIST %MS_EMOJI_FONT_PATH% (
     ECHO Creating new Segoe UI Emoji font from Twitter Color Emoji
-    ttx -t "name" -o "emjname.ttx" %MS_EMOJI_FONT_PATH% || GOTO :ERROR
-    ttx -o %FINAL_EMJ_FONT_PATH% -m %EMOJI_FONT_PATH% "emjname.ttx" || GOTO :ERROR
-    DEL "emjname.ttx"
+    call ttx -t "name" -o "emjname.ttx" %MS_EMOJI_FONT_PATH% || GOTO :ERROR
+    call ttx -o %FINAL_EMJ_FONT_PATH% -m %EMOJI_FONT_PATH% "%CD%\emjname.ttx" || GOTO :ERROR
+    DEL "%CD%\emjname.ttx"
```

Comment:
We need to know why or if the `%CD` is necessary, otherwise it should be removed

Reply:
`%cd%` means “current directory”. @Tenrys said that some commands were ignoring the `pushd`, so he added the `%cd%`s accordingly.

They could probably explain more about how it didn't work in their case, and the “before and after” results of `%cd%` additions.

Reply:
Here are two screenshots to illustrate why I added usage of `%CD%`.

`%CD%` absent:

![WindowsTerminal_Wgj0wx9C5K](https://github.com/13rac1/twemoji-color-font/assets/3979239/45242e9a-c479-4288-859c-8b4608451a7c)

`%CD%` present:

![WindowsTerminal_XjnoRmKWCi](https://github.com/13rac1/twemoji-color-font/assets/3979239/439b1d68-d248-4fe4-a661-47c29191c67c)

I don't know if the fact that I am calling `install.cmd` from PowerShell has any incidence on it, but it should make it more reliable in that case if it does.